bitnami/redmine/README.md

```diff
@@ -36,6 +36,91 @@ The command deploys Redmine on the Kubernetes cluster in the default configurati
 
 > **Tip**: List all releases using `helm list`
 
+## Deploying to a sub-URI
```

Comment:
Maybe this kind of context could fit better in [this section](https://github.com/bitnami/charts/tree/master/bitnami/redmine#configuration-and-installation-details), don't you think? Could you put it at the end of that section?

Reply:
Hello.

Agree ;)

I'll do that!